packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -17,32 +17,33 @@ import getParityWordlist from './utils/getParityWordlist';
 
 const debug = Debug('createAccountStore');
 
-export const SIGNER_ACCOUNTS_LS_KEY = `${LS_PREFIX}::paritySignerAccounts`;
 const DERIVATION_PATH = "m/44'/60'/0'/0/0";
-const MIN_PARITY_SIGNER_RECOVERY_WORDS = 11;
-const MAX_PARITY_SIGNER_RECOVERY_WORDS = 24;
 const ETHEREUM_ADDRESS_LENGTH = 40;
 const JSON_VERSION = 3;
+const MIN_PARITY_SIGNER_RECOVERY_WORDS = 11;
+const MAX_PARITY_SIGNER_RECOVERY_WORDS = 24;
+export const SIGNER_ACCOUNTS_LS_KEY = `${LS_PREFIX}::paritySignerAccounts`;
 
 export class CreateAccountStore {
   @observable
-  isImport = false; // Are we creating a new account, or importing via phrase?
+  address = null;
 
   @observable
-  jsonString = null;
+  bip39Phrase = null; // 12 to 24-words seed phrase
 
   @observable
-  parityPhrase = null; // 11 or 12-words seed phrase (Parity Signer used to generate a 11 words recovery phrase)
+  isImport = false; // Are we creating a new account, or importing via phrase?
 
   @observable
-  bip39Phrase = null; // 12 to 24-words seed phrase
+  jsonString = null;
 
   @observable
-  address = null;
+  name = ''; // Account name
 
   @observable
-  name = ''; // Account name
+  parityPhrase = null; // 11 or 12-words seed phrase (Parity Signer used to generate a 11 words recovery phrase)
```

Comment:
nitpick: ditto (not related to this PR)

Reply:
or even `... an 11-word ...`

Reply:
it keeps on getting better

Reply:
i've pushed commit 53a7c7e363b2cd5587e75923e5cf3222680b1fca